demos/common/cpp/models/src/detection_model_yolo.cpp

```diff
@@ -194,6 +204,43 @@ void ModelYolo3::parseYOLOV3Output(const std::string& output_name,
     }
 }
 
+// --------------------------- For YOLO with postprocess ------------------------------------------
+void ModelYolo3::parseYOLOV3Output(const InferenceEngine::Blob::Ptr& blob,
+    const unsigned long original_im_h, const unsigned long original_im_w,
+    std::vector<DetectedObject>& objects) {
+    const int detections_num = static_cast<int>(blob->getTensorDesc().getDims()[0]);
+    const int len_of_res = static_cast<int>(blob->getTensorDesc().getDims()[1]);
+    const float* output_blob = blob->buffer().as<PrecisionTrait<Precision::FP32>::value_type*>();
+
+    // --------------------------- Parsing YOLO output --------------------------------------------
+    for (int res_num = 0; res_num < detections_num; ++res_num) {
+        float object_score = output_blob[res_num * len_of_res + 4];
+        if (object_score < confidenceThreshold) {
+            continue;
+        }
+
+        for (int i = 5; i < len_of_res; ++i) {
```

Comment:
What is 5?

Reply:
The first five elements are coordinates and object score, elements sixth to n are probabilities. Five is the index of the element from which the probabilities start.